packages/osd-plugin-helpers/src/integration_tests/build.test.ts

```diff
@@ -94,8 +94,7 @@ it('builds a generated plugin into a viable archive', async () => {
   );
 
   expect(buildProc.all).toMatchInlineSnapshot(`
-    " warn These tools might work with 7.9 versions, but there are known workarounds required. See https://github.com/elastic/kibana/issues/82466 for more info
```

Comment:
Are we sure that we want to remove this warning and link?

Reply:
In packages/osd-plugin-helpers/src/cli.ts we removed version check so there is no log warning 'These tools might work with 7.9 versions, but there are known workarounds required'. That is the reason this particular functional test failed. Thanks for checking.

Reply:
this should be auto updated with `yarn run test:jest_integration -u`

Reply:
Yes. `yarn run test:jest_integration -u ` will generate same result by deleting that warning sentence. Mine also changed version 7.9.0 to 1.0.0, since we will change the versions eventually.